Sanitize request body during capturing instead of serialization #656 (#712)
elastic/apm-agent-dotnet: docs/configuration.asciidoc

This config accepts a list of wildcard patterns of field names which should be sanitized.
These apply for example to HTTP headers and `application/x-www-form-urlencoded` data.

> NOTE: In case you capture the request body manually through the public API, the agent won't sanitize the body - this setting only applies to values that are captured automatically by the agent.

Reviewer: Let's bump this from a NOTE to IMPORTANT. Also, I think it makes sense to swap the sentences. What do you think of this?
```suggestion
IMPORTANT: This setting only applies to values that are captured automatically by the agent. If you capture the request body manually with the public API, this configuration doesn't apply, and the agent won't sanitize the body. 
```

Author: Agreed 👍
